apps/website/src/pages/docs/variables.mdx

```diff
@@ -31,17 +31,17 @@ import '@itwin/itwinui-variables';
 
 The main background colors increase in darkness from 1 to 5.
```

Comment:
```suggestion
```

The line about 1 to 5 can be removed.

Reply:
You could change it to something like "There are several main background colors." to keep the format of "text then bullet points".